### PR TITLE
remove job's cluster kind enum validation and add slurm/lsf pipeline example

### PR DESCRIPTION
**Pull request by md-shabbir**

### CODE REVIEW

File: dkube/sdk/internal/dkube_api/models/job_cluster_model.py [hunk 1]
```diff
@@ -96,12 +96,17 @@ def kind(self, kind):
         :param kind: The kind of this JobClusterModel.  # noqa: E501
         :type: str
         """
+        # Shabbir: Manually commenting out the following piece of code to prevent raising exception because
```
Comment (mak-454) [inline dkube/sdk/internal/dkube_api/models/job_cluster_model.py:99]: can you add reference to 2.2.1.16 release here. And also mention that this change is to make sdk compatible with previous 2.1.x and 2.2.1.x release(s).

File: examples/python/parallel-pl.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import kfp.compiler as compiler
```
Comment (mak-454) [inline examples/python/parallel-pl.py:1]: you can remove this file.

File: examples/python/slurmjob.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+from dkube.sdk import *
```
Comment (mak-454) [inline examples/python/slurmjob.py:1]: you can delete this file

File: dkube/remote/job_properties.py [hunk 4]
```diff
@@ -2079,3 +2080,98 @@ def __ne__(self, other):
         """Returns true if both objects are not equal"""
         return not self == other
 
+
```
[the remark this answers is not in the document]
Comment (md-shabbir) [reply dkube/remote/job_properties.py]: addressed

**Comment by songole** [comment]
Where are you adding cluster CRUD APIs? @pallavi-pannu-oc is providing support for this in her pr #87 

@mak-454

**Comment by mak-454** [comment]
@songole this is for 2.2 and doesnt have operator APIs. pallavi raised her PR on main.

**Pull request merged**